spec/statement.dd

```diff
@@ -1163,14 +1163,31 @@ switch (name)
         less error prone. char, wchar and dchar strings are allowed.
         )
 
-        $(P $(D Implementation Note:) The compiler's code generator may
+        $(P If a $(GLINK CaseStatement), $(GLINK CaseRangeStatement), or $(GLINK DefaultStatement)
+        appears in the top level of the $(GLINK ScopeStatement), then a new scope for local
```

Comment:
According to -vcg-ast output, this happens for `case` in nested scopes, too, not only on top level.